frappe/public/js/frappe/desk.js

```diff
@@ -511,7 +511,8 @@ frappe.Application = class Application {
 		// 	"version": "12.2.0"
 		// }];
 
-		if (!Array.isArray(change_log) || !change_log.length || window.Cypress) {
+		if (!Array.isArray(change_log) || !change_log.length ||
+			window.Cypress || cint(frappe.boot.sysdefaults.disable_change_log_notification)) {
```

Comment:
@hrwX Correct me if I'm wrong, but it doesn't seem like all **System Settings** are automatically available in `frappe.boot.sysdefaults`.

Reply:
It is handled in `on_update` of [System Settings](https://github.com/frappe/frappe/blob/c31eca3ba589164d21174be2d578d66221007cdc/frappe/core/doctype/system_settings/system_settings.py#L46)